docs/reference/search/profile.asciidoc

```diff
@@ -646,18 +622,19 @@ And the response:
       }
 }
 --------------------------------------------------
+// TESTRESPONSE[s/\.\.\.//]
+// TESTRESPONSE[s/: (\-)?[0-9]+/: $body.$_path/]
+// TESTRESPONSE[s/: "[^"]*"/: $body.$_path/]
 
 As you can see, the output is significantly verbose from before.  All the major portions of the query are
 represented:
 
 1. The first `TermQuery` (message:search) represents the main `term` query
 2. The second `TermQuery` (my_field:foo) represents the `post_filter` query
-3. There is a  `MatchAllDocsQuery` (\*:*) query which is being executed as a second, distinct search.  This was
```

Comment:
The `MatchAllDocsQuery` does not appear in the original search query, neither in the updated one. I think the explanation is now obsolete but I'd be happy if someone can confirm/infirm this.

Reply:
This is kinda odd... I think we have to execute a secondary collector for the global agg (since the first is being filtered by the query).

I think modifying the docs right now is fine, but I'll start chasing why this changed because it could be something wrong with the profiler itself.